### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Toolbar_AI4D: Control type set wrong for add cell and toggle option 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#11195]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#WCAG4.1.2;#WCAG1.3.1;#SQL Azure Data Studio;#FTP;#AI4D;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: fd29e35
Date: 2020-07-02T05:40:02.292Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:WCAG4.1.2, WCAG1.3.1
Tool: Accessibility insights for desktop

**Repro Steps:**
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Right-click connection and click New Notebook
4. Click on cell and select code cell.
5.  Launch accessibility insights for desktop and hover over the tool bar .

**Actual:**
On hovering over the controls in the toolbar "add cell" and "toggle option" has been set as "menu item" and "list item" respectively.

**Expected:**
The control should be set as button and announced as button itself.

**User Impact:**
Screen reader users will get wrong control information.

**Attachment link for Reference:**
![11195A11y_AzureDataStudioStayHealthy_Notebook_Toolbar_AI4D](https://user-images.githubusercontent.com/24580570/86458553-0aceb980-bd43-11ea-89e8-a0517506d6d1.png)

**Comment by Charles-Gagnon** [comment]
@fsteffi We already set the aria role to button for these :

![image](https://user-images.githubusercontent.com/28519865/91357821-b37b2180-e7a6-11ea-92c8-77316a9ff842.png)

And they're announced correctly in NVDA :

```
tool bar
Run cell (F5)  button
tool bar
tool bar
Toggle More  button
Delete  button
Move cell up  button
Move cell down  button
Add cell  menu button  subMenu
```

So I don't see any issues there. Can you confirm if there's some other issue with our implementation?

@MaddyDev It does look like we're not correctly announcing the menu status for the "Toggle More" button though. Could you looking into fixing that?

**Comment by Roopesh-T** [comment]
the above issue is not repro in latest env so closing the bug.
![image](https://user-images.githubusercontent.com/86601127/220080635-9b2a1f62-fb82-4633-95e4-fe41fc9c3245.png)


![image](https://user-images.githubusercontent.com/86601127/220080118-6681f37e-dee4-408b-99b9-3e04f609ba9d.png)
![image](https://user-images.githubusercontent.com/86601127/220080539-d29306fb-612d-43ff-a10c-538a6ade76f6.png)
**Github Tag**:#Closed;
